pkg/brctl/util.go

```diff
@@ -154,7 +376,7 @@ func stringToJiffies(in string) (int, error) {
 	return int(tv.Seconds() * float64(hz)), nil
 }
 
-func readID(p string, obj string) (string, error) {
+func readString(p string, obj string) (string, error) {
```

Comment:
Imho, neither `readID` nor `readString` are descriptive names for this function. Maybe something like

```suggestion
func brctlReadSysfs(parent string, obj string) (string, error) {
```

Reply:
I was thinking of refactoring all those `readFoo()` functions into a more generic fashion, but did not find a smart solution quickly. However, taken that the similar functions are called `readInt()`, readBool()` etc. and given the places where the function is called, I think `readString` is okay here. Compared to the other functions it is just not necessary to do any casting. 

`readSysFS` would be perfect for a generic solution. Prefixing a function with "brctl" in a package "brctl" is not a good idea, though.